src/main/java/com/authservice/authservice/AuthServiceApplication.java

```diff
@@ -2,11 +2,18 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
 
 @SpringBootApplication
 public class AuthServiceApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(AuthServiceApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(AuthServiceApplication.class, args);
+    }
+
+    @Bean
+    public RestTemplate getRestTemplate() {
```

Comment:
Can you create a different file for all the @Bean initializations

Reply:
Done.